pybaseball/statcast_pitcher_spin.py

```diff
@@ -0,0 +1,140 @@
+from pybaseball import statcast_pitcher
+import pandas as pd
+import numpy as np
+
+K = .005153 #Environmental Constant
```

Comment:
would it be useful to compute this instead of hard-coding it? i.e. assume 70 F sea-level as a default but let a user specify something different in the call to `statcast_pitcher_spin`?

Reply:
As well, wouldn't this be park dependent? I.e., I'm assuming Tropicana Field (sea-level) vs Coors Field (a mile up) would possibly be different.

Reply:
It's great that you brought this up because I was hoping on replacing K with a calculation later. The reason I didnt in the original PR was to manage scope and keep the test data could be consistent.